lib/core/presentation/widgets/post/post_profile_card_widget.dart

```diff
@@ -99,7 +101,7 @@ class PostProfileCard extends StatelessWidget {
                 Text(
                   postText,
                   style: Typo.medium.copyWith(
-                    color: colorScheme.onPrimary,
+                    color: LemonColor.white87,
                     fontWeight: FontWeight.w400,
```

Comment:
Should use `colorScheme.onPrimary.withOpacity(0.87)`

Reply:
Ah I see 👍